src/main/java/org/demesup/controller/DepartmentController.java

```diff
@@ -0,0 +1,53 @@
+package org.demesup.controller;
+
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.demesup.model.Department;
+import org.demesup.model.field.DepartmentField;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.utils.Read.readPositiveNumber;
+import static org.utils.Utils.listInSeparatedLines;
+
+@Slf4j
+public class DepartmentController extends Controller {
+    @Override
+    public Department create() {
+        Department department = new Department();
+        Arrays.stream(DepartmentField.values()).forEach(f -> f.set(department));
+        repository.save(department);
+        return department;
+    }
+
+    @SneakyThrows
+    @Override
+    public void delete() {
+        delete(Department.class);
```

Comment:
Don't forget to add `super` when you call the super method. It gives more clarity